prettyprinter/src/Data/Text/Prettyprint/Doc/Internal.hs

```diff
@@ -1731,8 +1744,14 @@ layoutPretty
     :: LayoutOptions
     -> Doc ann
     -> SimpleDocStream ann
-layoutPretty = layoutWadlerLeijen
-    (FittingPredicate (\_pWidth _minNestingLevel maxWidth sdoc -> fits maxWidth sdoc))
+layoutPretty (LayoutOptions pageWidth_@(AvailablePerLine lineLength ribbonFraction)) =
+    layoutWadlerLeijen
+        (FittingPredicate
+             (\lineIndent currentColumn _initialIndentY sdoc ->
+                 fits
+                     (remainingWidth lineLength ribbonFraction lineIndent currentColumn)
+                     sdoc))
+        pageWidth_
```

Comment:
What’s this underscore for? Looks alien. Name conflict?

Reply:
`pageWidth` is the API name for the `WithPageWidth` constructor. Previously the variable was often called `pWidth` which I found hard to understand.